### ISSUE TITLE
Investigate UWP

### ISSUE THREAD

**Issue by dsafa** [dsafa/CSDeskBand#9]
I have not worked with UWP before but it should be doable. Microsoft has done it with their new "My People" app in the fall creators update.
![2017-12-28_20-57-25](https://user-images.githubusercontent.com/7334282/34429836-9d1c46ae-ec12-11e7-9e27-eb46c9be1bfe.png)
![inspect_2017-12-28_21-00-21](https://user-images.githubusercontent.com/7334282/34429838-9ec31a6e-ec12-11e7-8bb0-c227c6147b8d.png)

Taking a quick look, it is in fact a desk band so it can be something to investigate later.

**Comment by ibnuh** [comment]
Any updates to this? Looking forward to use CSDeskBand with UWP

**Comment by dsafa** [comment]
No, I never got far in the investigation. Maybe I'll take another look someday
